Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs

```diff
@@ -29,12 +29,12 @@
 using NUnit.Framework;
 using Realms;
 using System.ComponentModel;
-using AssemblyToProcess;
 using Mono.Cecil.Cil;
 
 namespace RealmWeaver
 {
     [TestFixture(WeaverOptions.RealmOnlyNoPropertyChanged)]
+    [TestFixture(WeaverOptions.PCLRealmOnlyNoPropertyChanged)]
```

Comment:
I thought about that. I guess we could, it just seemed a bit excessive to me but if you feel that it makes sense, I'll do it.